commit f6df4da0a66c9fdad6c1b794190431c265931a16
Author: Endi S. Dewata <edewata@redhat.com>
Date:   Thu May 31 04:34:48 2018 +0200

    Added .travis.yml
    
    A Travis CI configuration has been added to build JSS and run a
    basic installation test. Additional tests will be added later.

diff --git a/.travis.yml b/.travis.yml
new file mode 100644
index 00000000..ae674cc5
--- /dev/null
+++ b/.travis.yml
@@ -0,0 +1,22 @@
+# BEGIN COPYRIGHT BLOCK
+# (C) 2018 Red Hat, Inc.
+# All rights reserved.
+# END COPYRIGHT BLOCK
+
+services:
+  - docker
+
+install:
+  - docker pull registry.fedoraproject.org/fedora:28
+  - docker run
+      --name=container
+      --detach
+      -i
+      -v $(pwd):/root/jss
+      registry.fedoraproject.org/fedora:28
+  - docker exec container dnf install -y dnf-plugins-core gcc make rpm-build
+  - docker exec container dnf builddep -y --spec /root/jss/jss.spec.in
+  - docker exec container /root/jss/build.sh --with-timestamp --with-commit-id rpm
+
+script:
+  - docker exec container rpm -Uvh /root/build/jss/RPMS/*
